README.md

```diff
@@ -1,6 +1,6 @@
 # Solidus Internationalization
 
-[![Build Status](https://travis-ci.org/solidusio/solidus_i18n.svg?branch=master)](https://travis-ci.org/solidusio/solidus_i18n)
+[![CircleCI](https://circleci.com/gh/solidusio-contrib/solidus_gdpr.svg?style=svg)](https://circleci.com/gh/solidusio-contrib/solidus_gdpr)
```

Comment:
I think this should be `solidus_i18n` instead of `solidus_gdpr ` right?

Reply:
@spaghetticode fixed, sorry!